test/basic/ReportOnlyTest.gmk

```diff
@@ -40,6 +35,7 @@ $(BUILDTESTDIR)/ReportOnlyTest.ok: $(BUILDTESTDIR)/Basic.othervm.ok \
 		   $(JTREG_IMAGEDIR)/bin/jtreg
 	JAVA_HOME=$(JDKHOME) CLASSPATH=$(ABSCLASSDIR) \
 	    $(JTREG_IMAGEDIR)/bin/jtreg $(JTREG_OPTS) \
+	    -k:!needDisplay \
```

Comment:
This seems unrelated to the security manager changes.

Reply:
Sorry Jon, I forgot to reply to this comment. I'll refresh this PR against latest master branch and refresh my memory about this change and respond shortly.

Reply:
With the other PR https://github.com/openjdk/jtreg/pull/223 now integrated, this part of the change in this PR is no longer needed. So I've removed it in the latest updates to this PR.